lib/hexpm/repository/registry_builder.ex

```diff
@@ -183,7 +183,7 @@ defmodule Hexpm.Repository.RegistryBuilder do
     packages = Enum.map(packages, fn {name, _versions} -> %{name: name} end)
 
     %{packages: packages}
-    |> :hex_pb_names.encode_msg(:Names)
+    |> :hex_registry.encode_names()
```

Comment:
@ericmj we could even move building, encoding, signing and gzipping of registry protobufs to `hex_registry`.

See: [hexpm](https://github.com/hexpm/hexpm/blob/294f15e81329eb6fb3fc1247a4247e7544152a69/lib/hexpm/repository/registry_builder.ex#L182:L221) vs [mini_hex](https://github.com/wojtekmach/mini_hex/blob/master/lib/mini_hex/registry_builder.ex)

Happy to do that right now, or in a separate PR to hex_erl. Probably it's prudent to do that later :)

Reply:
Makes sense to move this but doing it later.